.github/workflows/meshery-cloud-release.yml

```diff
@@ -155,5 +155,5 @@ jobs:
           cd meshery-cloud
           sudo git pull
           sudo docker pull layer5/meshery-cloud:production-latest
-          sudo make prod-deploy
+          sudo make docker-prod-deploy
```

Comment:
I would like to see a deployment step included for cloud.layer5.io...

Reply:
Ok